roles/palava-signaltower/tasks/main.yml

```diff
@@ -27,5 +35,13 @@
 - name: Build Signaltower
   shell: "cd {{ palava_signaltower_install_dir }} && yes | mix deps.compile"
 
+- name: Install Signaltower SystemD unit
+  template:
+    src: signaltower.service.j2
+    dest: /etc/systemd/system/signaltower.service
+
 - name: Start Signaltower
-  shell: "{{ palava_signaltower_install_dir }}/start_daemon.sh"
\ No newline at end of file
+  systemd:
+    daemon_reload: yes
+    enabled: yes
+    name: signaltower.service
```

Comment:
Ok, two more steps are missing (that can be found on the current server in the post-deploy script of the tower):
- mix local.hex --force (this actually before mix deps.get, just to make sure this does not prevent the tower from starting)
- MIX_ENV=prod mix release production (to make a "release" which can then be found in $WorkingDirectory/_build/prod/rel/production)